src/js/components/StarRating/index.d.ts

```diff
@@ -1,7 +1,9 @@
 import * as React from 'react';
 import { RadioButtonGroupProps } from '../RadioButtonGroup/index';
 
-type RadioButtonGroup = Omit<RadioButtonGroupProps, 'options'>;
+type RadioButtonGroup = Omit<RadioButtonGroupProps, 'options'> & {
+  id?: string;
+};
```

Comment:
I think we can just import `RadioButtonGroupExtendedProps` instead of `RadioButtonGroupProps`. This way we don't need to explicitly define a type for `id`.

Reply:
Thanks for this info, i just updated as requested,